BankApp/Program.cs

```diff
@@ -0,0 +1,251 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace BankApp
+{
+    static class Program
+    {
+        private static List<Account> accounts = new List<Account>();
+        static void Main(string[] args)
+        {
+            do
+            {
+                Console.WriteLine();
+                Console.WriteLine("Choose action: ");
+                Console.WriteLine("Press 1 to create an account");
+                Console.WriteLine("Press 2 to make a domestic transfer");
+                Console.WriteLine("Press 3 to make an outgoing transfer");
+                Console.WriteLine("Press 4 to display your accounts\' balance");
+                Console.WriteLine("Press 5 to display transfers\' history");
+                Console.WriteLine("Press 0 to Exit");
+
+                int userChoice = GetIntFromUser("Select option");
+
+                switch (userChoice)
+                {
+                    case 0:
+                        return;
+                    case 1:
+                        CreateAccount();
+                        break;
+                    case 2:
+                        HandleDomesticTransfer();
+                        break;
+                    case 3:
+                        HandleOutgoingTransfer();
+                        break;
+                    case 4:
+                        PrintAccountsBalance();
+                        break;
+                    case 5:
+                        PrintAccountHistory();
+                        break;
+                    default:
+                        Console.WriteLine("Unknown option");
+                        break;
+                }
+            }
+            while (true);
+        }
+
+        private static void HandleDomesticTransfer()
```

Comment:
Looks like a lot of the logic of Dometic and Outgoing transfer is the same. Only the way receiver account is provided differs. I recommend moving common part of the methods onto a new method to avoid code duplications.